web/crux-ui/src/pages/teams/audit-log.tsx

```diff
@@ -58,18 +60,29 @@ const AuditLogPage = (props: AuditLogPageProps) => {
 
   const onShowInfoClick = (logEntry: AuditLog) => setShowInfo(logEntry)
 
+  useEffect(() => {
+    pagination.setItems(filters.filtered)
+  }, [filters])
+
   const selfLink: BreadcrumbLink = {
     name: t('common:audit'),
     url: ROUTE_TEAMS_AUDIT,
   }
 
-  const listHeaders = ['common:name', 'common:date', 'event', 'info'].map(it => t(it))
+  const pagination = usePagination<AuditLog>({
+    initialData: auditLog,
+    initialPagination: { pageSize: 10, currentPage: 0 },
+  })
+
+  const listHeaders = ['', 'common:name', 'common:date', 'event', 'info'].map(it => t(it))
```

Comment:
Why we need an empty item in the array?

Reply:
Because the first column (the placeholder avatar) doesn't have a header text just an empty cell.